packages/utils/test/matcher.spec.ts

```diff
@@ -60,7 +60,35 @@ describe('getMatchedPackagesSpec', () => {
     expect(getMatchedPackagesSpec('angular', packages).proxy).toMatch('google');
     // @ts-expect-error
     expect(getMatchedPackagesSpec('@fake/angular', packages).proxy).toMatch('npmjs');
+    // @ts-expect-error
     expect(getMatchedPackagesSpec('vue', packages)).toBeUndefined();
+    // @ts-expect-error
     expect(getMatchedPackagesSpec('@scope/vue', packages)).toBeUndefined();
   });
+
+  test('should return multiple uplinks in given order', () => {
+    const packages = {
+      react: {
+        access: 'admin',
+        publish: 'admin',
+        proxy: 'github npmjs',
+      },
+      angular: {
+        access: 'admin',
+        publish: 'admin',
+        proxy: 'npmjs gitlab',
+      },
+      '@fake/*': {
+        access: '$all',
+        publish: '$authenticated',
+        proxy: 'npmjs gitlab github',
+      },
+    };
+    // @ts-expect-error
+    expect(getMatchedPackagesSpec('react', packages).proxy).toMatch('github npmjs');
```

Comment:
I'm not 100% sure, but I recall there was some file referring the order matters, but I couldn't find it anymore.